Azure.ResourceManager.Core/ResourceGroupOperations.cs

```diff
@@ -9,43 +9,87 @@
 
 namespace Azure.ResourceManager.Core
 {
+    /// <summary>
+    /// A class representing the operations that can be performed over a specific ResourceGroup.
+    /// </summary>
     public class ResourceGroupOperations : ResourceOperationsBase<ResourceGroup>,
         ITaggableResource<ResourceGroup>, IDeletableResource
     {
+        /// <summary>
+        /// Gets initializes a new instance of the <see cref="ResourceType"/> class.
+        /// </summary>
         public static readonly ResourceType ResourceType = "Microsoft.Resources/resourceGroups";
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ResourceGroupOperations"/> class.
+        /// </summary>
+        /// <param name="options"> The client parameters to use in these operations. </param>
+        /// <param name="id"> The identifier of the resource that is the target of operations. </param>
         internal ResourceGroupOperations(AzureResourceManagerClientOptions options, ResourceIdentifier id)
             : base(options, id) { }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ResourceGroupOperations"/> class.
+        /// </summary>
+        /// <param name="options"> The client parameters to use in these operations. </param>
+        /// <param name="resource"> The resource that is the target of operations. </param>
         internal ResourceGroupOperations(AzureResourceManagerClientOptions options, Resource resource)
             : base(options, resource) { }
 
-        internal ResourceGroupsOperations Operations => GetClient<ResourcesManagementClient>((uri, creds) => new ResourcesManagementClient(
+        private ResourceGroupsOperations Operations => GetClient<ResourcesManagementClient>((uri, creds) => new ResourcesManagementClient(
             uri,
             Id.Subscription,
             creds,
             ClientOptions.Convert<ResourcesManagementClientOptions>()))?.ResourceGroups;
 
+        /// <summary>
+        /// Deletes a ResourceGroupOperations.
```

Comment:
Pull this text from the actual RP operations class
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/resources/Azure.ResourceManager.Resources/src/Generated/ResourceGroupsOperations.cs#L291